custom_components/petlibro/button.py

```diff
@@ -61,6 +61,14 @@ async def async_setup_entry(
 
     hub = entry.runtime_data
     _LOGGER.error(f"Found {len(hub.devices)} devices in the hub")
+
+
+    for device in hub.devices:
+        _LOGGER.error(f"Device type: {type(device)}, device name: {device.name}")
+        if isinstance(device, Feeder):
+            _LOGGER.error(f"Feeder device found: {device.serial}")
```

Comment:
Logger should use `%` and not fstring ([doc](https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html)).

```suggestion
        _LOGGER.error("Device type: %s, device name: %s", type(device), device.name)
        if isinstance(device, Feeder):
            _LOGGER.error("Feeder device found: %s", device.serial)
```